pkgs/development/misc/haskell/hasura/update.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p cabal2nix curl jq -I nixpkgs=.
+#
+# This script will update the hasura derivations to the latest version using
+# cabal2nix.
+#
+# Note that you should always try building hasura graphql-engine after updating it here, since
+# some of the overrides in pkgs/development/haskell/configuration-nix.nix may
+# need to be updated/changed.
+
+set -eo pipefail
+
+# This is the directory of this update.sh script.
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# graphql-engine derivation created with cabal2nix.
+engine_derivation_file="${script_dir}/graphql-engine.nix"
+parser_derivation_file="${script_dir}/graphql-parser.nix"
+ciinfo_derivation_file="${script_dir}/ci-info.nix"
+pgclient_derivation_file="${script_dir}/pg-client.nix"
+pool_derivation_file="${script_dir}/pool.nix"
+
+# TODO: get current revision of graphql-engine in Nixpkgs.
+# old_version="$(sed -En 's/.*\bversion = "(.*?)".*/\1/p' "$engine_derivation_file")"
+
+# This is the latest release version of graphql-engine on GitHub.
+new_version=$(curl --silent "https://api.github.com/repos/hasura/graphql-engine/releases" | jq '.[0].tag_name' --raw-output)
+
+echo "Running cabal2nix and outputting to ${engine_derivation_file}..."
+
+echo "# This has been automatically generated by the script" > "$engine_derivation_file"
+echo "# ./update.sh.  This should not be changed by hand." >> "$engine_derivation_file"
+
+# cabal2nix --revision "$new_version" --subpath server "https://github.com/hasura/graphql-engine.git" >> "$engine_derivation_file"
+cabal2nix --revision "v2.0.5" --subpath server --no-check "https://github.com/hasura/graphql-engine.git" >> "$engine_derivation_file"
```

Comment:
Could you add a comment in the code here explaining why `v2.0.5` is hard-coded on this line?  (And probably suggesting that it should be removed in the future)